### DISCUSSION TITLE
Error to import FMU from OpenModelica into Simulink

### DISCUSSION THREAD

**Discussion by Amuri44** [OpenModelica/OpenModelica#10657] [Q&A]
I have "OpenModelica v1.22.0-dev-73-gd238ab6c83 (32-bit)" Connected to "OMSimulator v2.1.1.post188-gaf996ad-mingw"

And I have an FMU mode generated by this script:
```modelica
>> buildModelFMU(FMU_Test, version="2.0", fmuType="cs",
              fileNamePrefix="outputtest", platforms={"dynamic"}, includeResources=false);
```
and the generated six files are:
.fmu
.txt
.libs
.txt
.makefile
.jbson

However, when I try to load the '.fmu' into Matlabl Simulink (via FMU Import in Simulink Extras Library), I got errors:

-Error parsing FMU 'pathtofile/FMUname.fmu'
-Caused by: Error in supplied FMU: Dynamic library file('FMUname.dll) does not exist in FMU 'pathtofile/FMUname.fmu'
Any way to solve it?

**Comment by lucas-mior** [comment]
You should check your FMU options (e.g OMEdit -> tools -> options).
From your error message, it seems that Simulink is expecting a dynamic library (.dll), and maybe your FMU is statically linked.
Which OS are you using?
